keyboards/keebio/iris/keymaps/osiris/keymap.c

```diff
@@ -0,0 +1,140 @@
+#include "iris.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
```

Comment:
Lines 13-16 should be changed to:

```c
  LOWER = SAFE_RANGE,
  RAISE,
```

as you aren't using the QWERTY or ADJUST keycodes, and you don't need them either.

Reply:
fixed